src/components/measure/MeasureDirective.js

```diff
@@ -43,7 +43,7 @@ goog.require('ga_measure_service');
                 update(evt.target);
               });
             });
-            newFeature.changed();
+            update(newFeature);
```

Comment:
@gjn no clue ... do you use the latest ol3 version? https://github.com/openlayers/ol3/releases/tag/v3.14.2